libraries/chain/block_header_state.cpp

```diff
@@ -297,7 +298,8 @@ void evaluate_finalizer_policies_for_promotion(const block_header_state& prev,
 
 void evaluate_proposer_policies_for_promotion(const block_header_state& prev,
                                               block_header_state& next) {
-   assert(next.timestamp() > prev.timestamp()); // next block timestamp must be greater than nextent timestamp
+   // next block timestamp must be greater than next timestamp, validated in get_active_proposer_policy_for_block_at
+   assert(next.timestamp() > prev.timestamp());
```

Comment:
EOS_ASSERT?

Reply:
Validated in get_active_proposer_policy_for_block_at, so it can remain an `assert` here.